src/components/NewMessageModal.vue

```diff
@@ -87,6 +89,7 @@ export default {
 	},
 	data() {
 		return {
+			toolbar: undefined,
```

Comment:
```suggestion
			toolbarElements: undefined,
```

to hint that it's an array